src/index.ts

```diff
@@ -15,3 +15,5 @@
 export { GoogleAnalytics } from './third-parties/google-analytics';
 export { GoogleMapsEmbed } from './third-parties/google-maps-embed';
 export { YouTubeEmbed } from './third-parties/youtube-embed';
+
+export * from './types';
```

Comment:
just a way to get the types exported...

Reply:
My TS skills are not as good as I'd like them to be lol

Is this how folks generally export types? Do you know if packages usually includes a types file as well?

Reply:
there's maybe a more "proper" way, this just made it work for me. 
@janicklas-ralph - what did you have in mind? 

would be nice maybe to be able to import them from "third-party-capital/types" instead?

Reply:
Yeah tbh I'm not sure, but we can figure that out in a future PR :)